kubernetes/utils/create_from_yaml.py

```diff
@@ -30,7 +31,7 @@ def create_from_yaml(
     Perform an action from a yaml file. Pass True for verbose to
     print confirmation information.
     Input:
-    yaml_file: string. Contains the path to yaml file.
+    yaml_file: string. Contains yaml or a  path to yaml file.
```

Comment:
This is a little ambiguous. Does this mean yaml string, parsed yaml dict, or yaml file?
I think we should say "contains a yaml string, path to yaml file, or a dict object" or something like that

Reply:
Thanks for the suggestion. I fixed this.